### ISSUE TITLE
Relaxed is not accepting + character

### ISSUE THREAD

**Issue by akefeli** [afair/email_address#82]
Docs mention "Relaxed - A less strict form of Conventional, same character set", so I was assuming it would be a superset of conventional, but

```
irb(main):086:0> EmailAddress.valid? 'foo+abc@example.com', host_validation: :syntax, local_format: :conventional
=> true
irb(main):087:0> EmailAddress.valid? 'foo+abc@example.com', host_validation: :syntax, local_format: :relaxed
=> false
```

Is this expected?

**Comment by akefeli** [comment]
Am I missing something or is this code missing to return `true` when tag exists?

    def relaxed?
      self.syntax = :invalid
      self.valid_size? or return false
      self.valid_encoding? or return false
      if self.tag
        return false unless self.mailbox =~ RELAXED_MAILBOX_REGEX &&
          self.tag =~ RELAXED_TAG_REGEX
      elsif self.local =~ RELAXED_MAILBOX_REGEX
        self.syntax = :relaxed
        true
      else
        false
      end
    end

**Comment by afair** [comment]
You were right, the code was incorrect! Thanks for bringing this up. Released as v0.2.1
